docs/_posts/examples/3400-01-10-live-update-feature.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: example
+category: example
+title: Update a feature in realtime
+description: Change an existing feature on your map in realtime by updating its data
+tags:
+  - sources
+---
+<div id='map'></div>
+
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/satellite-v9',
+    zoom: 0
+});
+
+var url = '{{site.baseurl}}/assets/hike.geojson';
+map.on('load', function () {
+    var xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = function() {
```

Comment:
Hmm, I saw that, but I thought it better to stay to conventional, other-dependency-free JS.